jest.config.js

```diff
@@ -2,4 +2,7 @@
 
 module.exports = {
   setupFiles: ['./test/setup.js'],
+  testMatch: [
```

Comment:
do we need this?

Reply:
by default jest will look at `[ '**/__tests__/**/*.js?(x)', '**/?(*.)(spec|test).js?(x)' ]` we can narrow it down. I am not certain if it is perf improvement for test but  I have seen this config in most projects.